labelbox/orm/model.py

```diff
@@ -54,6 +54,17 @@ def __init__(self, enum_cls: type):
         def name(self):
             return self.enum_cls.__name__
 
+    class ListType:
+
+        def __init__(self, list_cls: type):
+            self.list_cls = list_cls
+
+        @property
+        def name(self):
+            if self.list_cls.__name__ == "DataRowMetadataField":
+                return "[DataRowCustomMetadataUpsertInput!]"
+            return self.list_cls.__name__
```

Comment:
Could this also just be a param of the ListType? This could be the param_name that is set in the field declaration in the DataRow class?

Reply:
Done!